src/rpk_entity.rs

```diff
@@ -0,0 +1,85 @@
+use crate::error::Error;
+use crate::signed_data::SubjectPublicKeyInfo;
+use crate::{der, signed_data, DerTypeId};
+use pki_types::{SignatureVerificationAlgorithm, SubjectPublicKeyInfoDer};
+
+/// A Raw Public Key certificate, used for connections using raw public keys as specified in RFC7250.
```

Comment:
```suggestion
/// A Raw Public Key, used for connections using raw public keys as specified in RFC7250.
```
note raw public keys are not certificates, nothing certifies anything

Reply:
Ah of course! Just updated it everywhere